app/ota_status.py

```diff
@@ -21,25 +18,17 @@ class OtaStatus(Enum):
     ROLLBACK_FAILURE = 5
 
 
-class OtaStatusControl:
+class OtaStatusControlMixin:
     def __init__(self):
-        self._ota_partition = OtaPartitionFile()
-        self._ota_status = self._initialize_ota_status()
-        logger.info(f"ota_status={self._ota_status}")
+        # ota_status will be initialized by ota_client
+        self._ota_status: OtaStatus = None
```

Comment:
I think this attribute initialization is not required since this class is mixin.

Reply:
The same as other mixin, this __init__ method is just a placeholder, we don't need to call it.
_ota_status will be initialized in the OtaClient's __init__ method.

Reply:
@Bodong-Yang 
https://stackoverflow.com/questions/36690588/should-mixins-use-parent-attributes
Mixin class should use base class attribute.

Reply:
> 
yes I am using base class attribute here.

Reply:
base class is initialzing this attribute, so this __init__ function should be removed from mixin class I think.